SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -566,6 +567,17 @@ void FSpatialGDKEditorToolbarModule::VerifyAndStartDeployment()
 		return;
 	}
 
+	if (bSchemaBuildError)
+	{
+		UE_LOG(LogSpatialGDKEditorToolbar, Warning, TEXT("Schema did not previously compile correctly, you may be running a stale build."));
+
+		EAppReturnType::Type result = FMessageDialog::Open(EAppMsgType::YesNo, FText::FromString("Last schema generation failed or failed to run the schema descriptor. Schema will most likely be out of date and cause a crash. Are you sure you want to continue?"));
```

Comment:
```suggestion
		EAppReturnType::Type Result = FMessageDialog::Open(EAppMsgType::YesNo, FText::FromString("Last schema generation failed or failed to run the schema descriptor. Schema will most likely be out of date, which may lead to undefined behavior. Are you sure you want to continue?"));
```

Reply:
Supposedly the main places where the game crashed because of out of date schema was fixed. If that isn't true we should create tickets for those (unless they're already created).

Reply:
I am working on another bug which will show an message when the schema does not match (a checksum/hash) - this might cover all those situations? 

The UNR does suggest gracefully exiting on those situations but i'm not sure :D